### PR TITLE
Updated Arabic README

### PR DESCRIPTION
**Pull request by pewpewded**
### Questions
- [x] Did you test your changes or double check that they work?
- [x] Did you read and follow the [Atlas Contribution Guidelines](https://docs.atlasos.net/contributions/)?
- [ ] Did you commit to the [`dev`](https://github.com/Atlas-OS/Atlas/tree/dev) branch and not [`main`](https://github.com/Atlas-OS/Atlas)?

**Note:** You should commit directly to `main` for translations of the `README.md`.

### Describe your pull request

This pull request updates the Arabic README translation with the latest changes 👍🏻

* Added the translations section, I genuinely think it looks better with it, regardless of whether it is useful or not.
  * also added the US flag to the pack, it routes to `/Atlas#readme`

### CODE REVIEW

File: translations/README_ar_SA.md [hunk 1]
```diff
@@ -15,9 +28,39 @@
...
+<kbd>[<img title="Türkçe" alt="Türkçe" src="https://gcore.jsdelivr.net/gh/hampusborgos/country-flags@main/svg/tr.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_tr_TR.md)</kbd>
+<kbd>[<img title="українська мова" alt="українська мова" src="https://gcore.jsdelivr.net/gh/hampusborgos/country-flags@main/svg/ua.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_ua_UA.md)</kbd>
+<kbd>[<img title="Tiếng Việt" alt="Tiếng Việt" src="https://gcore.jsdelivr.net/gh/hampusborgos/country-flags@main/svg/vn.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_vi_VN.md)</kbd>
+<kbd>[<img title="中文" alt="中文" src="https://gcore.jsdelivr.net/gh/hampusborgos/country-flags@main/svg/cn.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_zh_CN.md)</kbd>
+
+</div>
+
+#### _هل تريد مساعدتنا بترجمة المستندات؟ إذا كان الأمر كذلك ، يرجى اتباع هذه [التعليمات](https://github.com/Atlas-OS/Atlas/blob/main/translations/README.md)._
+
```
Comment (PencilNavigator) [inline translations/README_ar_SA.md:60]: You do not add the flag changer here. Follow the format [here](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_zh_CN.md) as a reference.

File: translations/README_ar_SA.md [hunk 2]
```diff
@@ -1,7 +1,20 @@
-﻿⚠️Note: This is a translated version of the original [README.md](https://github.com/Atlas-OS/Atlas/blob/main/README.md), information here may not be accurate and can be outdated.
```
Comment (PencilNavigator) [inline translations/README_ar_SA.md:1]: you do not delete this. Follow the [Translation Guidelines](https://github.com/Atlas-OS/Atlas/blob/main/translations/README.md).

File: translations/README_ar_SA.md [hunk 3]
```diff
@@ -95,19 +148,21 @@
...
+[جَرّب العبث بعلامتنا التجاريّة واصنع شيئًا مُذهلًا!](https://cdn.jsdelivr.net/gh/Atlas-OS/Atlas@main/img/brand-kit.zip)
 
-أيضًا لدينا [مساحة خاصّة للمبدعين](https://forum.atlasos.net/t/art-showcase), حتّى تتمكّن من مُشاركة أفكارك وإبداعاتك مع عباقرة ومبدعين آخرين!
+أيضًا لدينا [مساحة خاصّة للمبدعين](https://forum.atlasos.net/t/art-showcase)، حتّى تتمكّن من مُشاركة أفكارك وإبداعاتك مع عباقرة ومبدعين آخرين!
 
 ## ⚠️ إخلاء مسؤوليّة - Disclaimer
 
 <div align='right'>
 
-https://github.com/Atlas-OS/Atlas#%EF%B8%8F-disclaimer
```
Comment (PencilNavigator) [inline translations/README_ar_SA.md:110]: keep the old link,

**Review by PencilNavigator — CHANGES_REQUESTED** [review]
Follow Translation Guidelines fully and follow the format [here](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_zh_CN.md) as a reference.

File: translations/README_ar_SA.md (hunk 1 above)
Comment (pewpewded) [reply translations/README_ar_SA.md:60]: deleted

File: translations/README_ar_SA.md (hunk 2 above)
Comment (pewpewded) [reply translations/README_ar_SA.md:1]: reverted

File: translations/README_ar_SA.md (hunk 3 above)
Comment (pewpewded) [reply translations/README_ar_SA.md:110]: it's technically the same link, but as you wish.

**Comment by pewpewded** [comment]
I didn't realize you fixed them when i wrote the comments :D

Good Luck!

**Comment by PencilNavigator** [comment]
it's merged https://github.com/Atlas-OS/Atlas/blob/main/translations/README_hi_HI.md
thx for the awesome work!

**Comment by pewpewded** [comment]
that's not mine it': https://github.com/Atlas-OS/Atlas/blob/main/translations/README_ar_SA.md

there's minor issues to be fixed i'll fork and pr.

no need! keep up the good work y'all, always happy to contribute to Atlas.

**Pull request merged**